src/pybind/mgr/orchestrator.py

```diff
@@ -119,7 +119,7 @@ def copy_to_this_subinterpreter(r_obj):
         # This is something like `return pickle.loads(pickle.dumps(r_obj))`
         # Without importing anything.
         r_cls = r_obj.__class__
-        if r_cls.__module__ == '__builtin__':
+        if r_cls.__module__ in ('__builtin__', 'builtins'):
             return r_obj
         my_cls = getattr(sys.modules[r_cls.__module__], r_cls.__name__)
         if id(my_cls) == id(r_cls):
```

Comment:
Any suggestions to test this?
It seems if using `Exception()` as `r_obj`, it will be returned on line 126 too without the change.

Reply:
Fixed and added a test.

`id(my_cls) == id(r_cls):` is false, because of subinterpreter witchcraft.